frontend/src/app/application-forms/validators/phone-number-max-length-validation.ts

```diff
@@ -0,0 +1,18 @@
+import { FormControl, ValidatorFn } from '@angular/forms';
+
+export function phoneNumberMaxValidator(): ValidatorFn {
+  return (control: FormControl): { [key: string]: any } => {
+    const val = control.value;
+    if (val && val.length) {
+      const phoneNmbrRegex = /^[0-9-]{2,}[0-9]$/;
```

Comment:
No biggie but I'd rather just have `Number` spelled out. We got the CPU cycles to spare on two characters in exchange for readability :)

Reply:
@Dmac26 ^^